controller/lib/core/sensors.cpp

```diff
@@ -142,14 +117,21 @@ float get_pressure_reading(AnalogPinId pinId) {
   return ((float)runningSum * ADC_LSB);
 }
 
-float convert_diff_pressure_to_volumetric_flow(
-    float diffPressureInKiloPascals) {
-  if (diffPressureInKiloPascals > 0.0f) {
-    return ROOT_OF_TWO_OVER_DENSITY_OF_AIR * venturiAreaProduct *
-           bernoulliAreaDivisor * sqrtf(diffPressureInKiloPascals * 1000.0f);
-  } else {
-    return -1.0f * ROOT_OF_TWO_OVER_DENSITY_OF_AIR * venturiAreaProduct *
-           bernoulliAreaDivisor *
-           sqrtf(-1.0f * diffPressureInKiloPascals * 1000.0f);
-  }
+float pressure_delta_to_volumetric_flow(float diffPressureInKiloPascals) {
+  // PortArea must be larger than the ChokeArea [meters^2]
+  float venturiPortArea =
```

Comment:
Can you leave a ```TODO(jlebar): Make these contstexpr once we have a C++ standard library```?  That would ensure they're compile-time constants.  But we need `std::sqrt` and friends first.  I think that may be possible once we have STM32.

Reply:
I agree